components/HomeHero/HomeHero.module.scss

```diff
@@ -0,0 +1,59 @@
+.title {
+    @include font-size(xxxxl);
+    margin: 0;
+    line-height: 1;
+    letter-spacing: -1px;
+    font-weight: $weight--extrabold;
+
+    @include media-query(large) {
+        @include font-size(xxxxxl);
+    }
+
+    &__emphasis {
+        color: var(--color--accent);
+        font-weight: $weight--extrabold;
+    }
+}
+
+.logo {
+    max-width: 128px;
+    max-height: 135px;
+
+    @include media-query(medium) {
+        max-width: 171px;
+        max-height: 180px;
+    }
+
+    @include media-query(large) {
+        align-self: flex-end;
+        margin-right: 1.5rem;
+        max-width: 213px;
+        max-height: 225px;
+    }
+}
+
+.hero {
+    margin-top: 4rem;
+    max-width: 100%;
+
+    @include media-query(large) {
+        margin-top: 7.5rem;
+        display: flex;
+        margin-bottom: 4rem;
+    }
+
+    &__body {
+        position: relative;
+        top: -45px;
+        left: -5px;
+
+        @include media-query(medium) {
+            top: -50px;
+        }
+
+        @include media-query(large) {
+            top: 0;
+            left: 0;
+        }
+    }
```

Comment:
Careful with absolute positioning here: the strapline component that is a child of the body runs outside of the page container when the window is 1000px wide.

<img width="1056" alt="image" src="https://user-images.githubusercontent.com/18164832/154986999-18a79cea-1a40-420f-a124-698805f46dd4.png">

Reply:
None of the strapline component and the logo are absolutely positionned.

The strapline component running outside of the page container is due to the `flex-shrink: 0;` rule. Increasing the `large` breakpoint from 1023px to 1132px fixes that. However I don't know if that's the best solution.

Reply:
Sorry, I was confused with the relative positioning - flex-shrink is indeed the issue, I don't often use this. The other solution that thought of for this would be to make all the content `inline` on desktop - something like the following. 
![image](https://user-images.githubusercontent.com/18164832/155372356-69589c7d-d20d-4917-be2c-89ca3747d5b0.png)
The `.strapline` class would no longer display as flex on media query large to properly realign the text.

Reply:
I tried following your suggestion but I was still facing the issue. What do you think of my suggestion of increasing the large breakpoint value?

Reply:
I increased the large breakpoint value to avoid this issue.